### ISSUE TITLE
Command inputs don't work

### ISSUE THREAD

**Issue by joe-tofu** [moranje/alfred-workflow-todoist#24]
On a fresh install of v4.0.2 (Alfred v3.5.1) a lot of central commands don't seem to work. 

The "todo"-command doesn't do anything, "todos" get's listed in Alfred, but searching doesn't work.The "t"-command get's listed in Alfred, but only ":update" works. Neither ":token", nor ":language" or ":nodejs" works. 

Debugging only gives back the following – (at least to me) not very informative – errors:

For inputting the "todo"-command (without any following text):
`[2017-11-08 21:06:12][ERROR: input.scriptfilter] Code 127: chmod: build: No such file or directory
/Users/jmayer/Library/Caches/com.runningwithcrayons.Alfred-3/Workflow Scripts/B78E5FE2-06E0-4C32-9159-4FA06B4ABE27: line 15: node: command not found
[2017-11-08 21:06:12][ERROR: input.scriptfilter] Code 127: /bin/bash: line 9: node: command not found`

For inputting the "t:token"-command:
`[2017-11-08 21:07:16][ERROR: input.scriptfilter] Code 127: chmod: build: No such file or directory
/Users/jmayer/Library/Caches/com.runningwithcrayons.Alfred-3/Workflow Scripts/F5219386-C6A8-433E-A9F8-4B886DFC44AB: line 15: node: command not found
[2017-11-08 21:07:16][ERROR: input.scriptfilter] Code 127: /bin/bash: line 14: node: command not found
[2017-11-08 21:07:16][ERROR: input.scriptfilter] Code 127: /bin/bash: line 14: node: command not found
[2017-11-08 21:07:16][ERROR: input.scriptfilter] Code 127: /bin/bash: line 14: node: command not found
[2017-11-08 21:07:17][ERROR: input.scriptfilter] Code 127: /bin/bash: line 14: node: command not found
[2017-11-08 21:07:17][ERROR: input.scriptfilter] Code 127: /bin/bash: line 14: node: command not found
[2017-11-08 21:07:17][ERROR: input.scriptfilter] Code 127: /bin/bash: line 14: node: command not found`

I don't know if this means anything (I have absolutely no clue how Alfred workflows work), but comparing the src with the contents of the installed workflow I found, that there are certain differences. For example the "shell"-folder only contains three .sh-files, while the source contains eight.

<img width="243" alt="bildschirmfoto 2017-11-08 um 21 12 08" src="https://user-images.githubusercontent.com/3831916/32571978-8768129a-c4c9-11e7-8384-50185ccc949c.png">

**Comment by mikedett91** [comment]
@joe-tofu I do not think it is an issue with the workflow, I'm using it with the versions that you posted. Can you post the command you used to update your token? And what is the behavior you see? "Does not work" only tells so much :)

**Comment by smagdali** [comment]
I'm seeing this too. I have installed node.js (and if I try and install again it complains), but not all commands and none of the fancy parsing features are working for me. Running 3.14

**Comment by smagdali** [comment]
fixed by upgrading to 4.02
